realm/realm-library/src/main/cpp/CMakeLists.txt

```diff
@@ -162,13 +162,12 @@ file(GLOB objectstore_SRC
     "object-store/src/impl/collection_change_builder.cpp"
     "object-store/src/impl/transact_log_handler.cpp"
     "object-store/src/impl/weak_realm_notifier.cpp"
-    "object-store/src/impl/android/*.cpp"
-    "object-store/src/util/*.cpp")
+    "object-store/src/util/*.cpp"
+    "object-store/src/impl/epoll/*.cpp"
+    "object-store/src/util/android/*.cpp")
 # Sync needed Object Store files
 if (build_SYNC)
-    file(GLOB objectstore_sync_SRC
-        "object-store/src/sync_manager.cpp"
-        "object-store/src/sync_session.cpp")
+    file(GLOB objectstore_sync_SRC "object-store/src/sync/*")
```

Comment:
We don't actually need anything from that directory. Add it in your SyncManager PR later ?